src/js/components/action.jsx

```diff
@@ -17,6 +18,18 @@ export class ActionComponent extends Component {
         };
     }
 
+    onPostbackClick = () => {
+        this.setState({
+            state: 'processing'
+        });
+
+        postPostback(this.props._id).then(() => {
```

Comment:
Actually, we do more than that, we show an error notification as well: https://github.com/smooch/smooch-js/blob/f/postback/src/js/services/stripe-service.js#L8